src/services/utils.ts

```diff
@@ -182,4 +182,23 @@ export class Utils {
     documentElement?.style.setProperty(varName, value);
   }
 
+  // @TODO discuss using library like lodash or underscore
```

Comment:
I'd say let's use tools, that make us develop faster.
After that worry about bundling and performance.

So +1 for lodash (over underscore)